src/Renderer/Label2DRenderer.js

```diff
@@ -8,7 +8,7 @@ function isIntersectedOrOverlaped(a, b) {
 // find label in children
 function hasLabelChildren(object) {
     const parent = object.parent;
-    return parent.material && !parent.material.visible &&
+    return !(parent.material?.visible) &&
```

Comment:
This line will not have the same value as before if `parent.material` is `undefined` : previously, it would have returned `undefined` and it now returns `true`.

Reply:
done